src/azure-cli-core/azure/cli/core/aaz/_operation.py

```diff
@@ -204,7 +204,7 @@ def processor(schema, result):
         else:
             data = value
 
-        if data == AAZUndefined or data == None:  # noqa: E711, pylint: disable=singleton-comparison
+        if data in (AAZUndefined, None):  # noqa: E711, pylint: disable=singleton-comparison
```

Comment:
Please remove the comment to silence E711 and pylint.

Reply:
```suggestion
        if data in (AAZUndefined, None):
```